samples/caf_sensor_manager/remote/boards/nrf54h20dk_nrf54h20_cpuppr.overlay

```diff
@@ -5,10 +5,6 @@
  */
 
  / {
-	chosen {
```

Comment:
Because it is now set in the [board definition](https://github.com/nrfconnect/sdk-zephyr/pull/1577/files#diff-36d62416eda9833ee558e0a7b39226885a8fba2a2b4955d434174975db08bfb8R28).

Reply:
is `cpuppr_code_data` pointing to `cpuppr_ram3x_region` by default or do you need to define `cpuppr_code_data` in every sample?
This sample shrank the PPR partition in RAM3 for internal purposes.

Reply:
`cpuppr_code_data` is now part of the [default memory map](https://github.com/nrfconnect/sdk-zephyr/pull/1577/files#diff-eacbc217e4cb5d0e9199d37b2a623b24e0cab0fd7758e1274aef64b9fd1a5a20R110). Previously, `cpuppr_ram3x_region` wasn't divided into multiple sub-regions, so that had to be done for this sample, but not anymore.